src/ghci/mod.rs

```diff
@@ -370,6 +378,11 @@ impl Ghci {
                 .await?;
         }
 
+        for command in &self.opts.hooks.after_reload_ghci {
```

Comment:
should this be slightly further up (inside the `if !actions.needs_reload.is_empty() {` expression)?